src/pkg/selectors/sharepoint.go

```diff
@@ -213,6 +213,21 @@ func (s *sharePoint) Sites(sites []string) []SharePointScope {
 	return scopes
 }
 
+// Lists produces one or more SharePoint list scopes.
```

Comment:
I can't leave a comment up that far, but `Sites` and `WebURL` need additional `makeScope` calls for Lists.  You can copy the current makeScope and swap library for list.

Reply:
`Sites` and `WebURL` adjusted with  commit [0ed968f](https://github.com/alcionai/corso/pull/1786/commits/0ed968fec8fdfe3ceadd78b6a6422459c89fc133)

Reply:
> Edit: this goes for other places, too. setDefaults on ~450 (418 in the original) and reduce below that. Basically, look for anywhere that SharePointLibrary or SharePointLibraryItem is used in this file, to see if it needs an addition for a List.

Updates made within [40cf373](https://github.com/alcionai/corso/pull/1786/commits/40cf3735de421da22bf5959fd2937694ecfb7483)

Reply:
Last remaining bit: the `pathValues` func needs a list entry.  You might want to look at the selectors/exchange.go version for boilerplate on a multi-category version of that func.

Reply:
PathValues updated in [75fd226](https://github.com/alcionai/corso/pull/1786/commits/75fd226901b473be270c64f013bc688ab38aea25). 
Tests will have to be adjusted in the following commits.

Reply:
Unit tests updated in [ae1a184](https://github.com/alcionai/corso/pull/1786/commits/ae1a184ae0c1e449f8a9e68d903b0aa38c70360b)